docs/hugo/content/introduction/authentication.md

```diff
@@ -15,12 +16,18 @@ Azure Service Operator supports two different styles of authentication today.
 
 To use Service Principal authentication, specify an `aso-controller-settings` secret with `AZURE_CLIENT_ID` and `AZURE_CLIENT_SECRET` set.
```

Comment:
Do we need to explicitly call out that these are environment variables?

Reply:
They're not. They're secret keys. Have updated wording some to hopefully clarify this.